src/osgOcean/OceanScene.cpp

```diff
@@ -21,6 +21,10 @@
 
 #include <osg/Depth>
 
+
+#define SCREEN_WIDTH 1920

```

Comment:
If it is a parameter, why not simply making it a parameter ????? Instead of hardcoding a different value for your own needs.